integration/spark/shared/src/main/java/io/openlineage/spark/agent/util/DatasetIdentifier.java

```diff
@@ -1,44 +0,0 @@
-/*
-/* Copyright 2018-2024 contributors to the OpenLineage project
-/* SPDX-License-Identifier: Apache-2.0
-*/
-
-package io.openlineage.spark.agent.util;
-
-import java.util.LinkedList;
-import java.util.List;
-import lombok.Value;
-
-@Value
-public class DatasetIdentifier {
```

Comment:
Duplicate?

Reply:
this got moved to java client some time ago